manifests/03_credentials_request_azure.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     include.release.openshift.io/single-node-developer: "true"
     capability.openshift.io/name: Storage
 spec:
+  serviceAccountNames:
+  - azure-disk-csi-driver-operator
+  - azure-disk-csi-driver-controller-sa
```

Comment:
@jstuever I think we need one more SA here: `azure-disk-csi-driver-node-sa`

Because nodes need to authenticate too in order to mount volumes, on the other hand we probably don't need the operator SA at all.